draft-ietf-tls-esni.md

```diff
@@ -116,6 +116,27 @@ document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here. All TLS
 notation comes from {{RFC8446, Section 3}}.
 
+# Terminology
+
+The following terms are defined in **this** document:
+
+* Backend server: the origin server.
+
+* Client-Facing server: the entity trusted to update the ECH configuration.
```

Comment:
These terms, as well as "ECH-service provider", already have a definition in the "Topologies" section, which is part of the larger "Overview" section:
```
In the remainder of this document, we will refer to the ECH-service provider as the "client-facing server" and to
the TLS terminator as the "backend server". These are the same entity in Shared Mode, but in Split Mode, the
client-facing and backend servers are physically separated.
```

Reply:
Well, no there are at least 3 terms 
provider
ECH-service provider
client-facing server

This is not coherent and add to the general confusion. I would have been tempted to even kill the 'provider' approach but it makes sense pedagogically so at least to be coherent between either 'provider' or 'ECH-service provider' 

I prefer explicit so I prefer ECH-service provider and like wise I will keep 'public name' but I find it really a dangerous abuse of words in the text .. when you read the first time the text how many times did I ask myself but this is the public name of what ... and the abuse of public name vs public_name, I understand the approach and it has some elegancy but boy ... again if you are not in ECH since years, think a bit about operational people who will have to swallow this one.